resources/translations/fr_FR/about/preferences.md

```diff
@@ -0,0 +1,11 @@
+[//]: # (Do not translate this file! While translating the main text doesn't cause any harm, translating the labels of the preferences here will cause them to stop working.)
+Preferences
+====
+In this page you can change the behaviour of the settings guide.
```

Comment:
Did you intend to translate this part?

It's the translation of the lines that start with `[ ]` that causes problems. The preferences key that the checkboxes change is generated from the human-readable text.

Reply:
Yes , I think it's nice also in the settings to have a translated file in order to get in the langage choice drop down menu to have also the choice of the langage. And as it was already the case for the german translation i thought it could be a standard to translate this file as well

Reply:
Yeah fine. I added that to the whole file to keep it simple. But if you understand the details, some parts of it can indeed be translated.

Currently it just adds an English copy though. None of this is French yet. Did you intend to translate it?